tools/clang/include/clang/SPIRV/SpirvType.h

```diff
@@ -279,7 +320,9 @@ class SpirvPointerType : public SpirvType {
 
   static bool classof(const SpirvType *t) { return t->getKind() == TK_Pointer; }
 
-  const SpirvType *getPointeeType() const { return pointeeType; }
+  const SpirvType *getPointeeType() const {
+    return llvm::cast<SpirvType>(pointeeType);
```

Comment:
No need to cast? Applies to other places.

Reply:
Done